flaml/tune/tune.py

```diff
@@ -448,6 +448,7 @@ def easy_objective(config):
             logger.addHandler(old_handlers[0])
         if verbose > 0:
             if log_file_name:
+                os.makedirs(os.path.dirname(log_file_name), exist_ok=True)

```

Comment:
I do think it's better to use `pathlib` for these sorts of operations, but it's nothing crucial.

Reply:
Thanks. Feel free to propose changes or make PRs about it.